pallets/attesters/src/lib.rs

```diff
@@ -267,8 +356,7 @@ pub mod pallet {
             origin: OriginFor<T>,
             message: Vec<u8>,
```

Comment:
Typically in signature schemes the message is a hash of the data that is signed. Makes it a lot faster + in this case less call data. The hash can then also act as the batch id